nvOC_miner_update.sh

```diff
@@ -385,16 +785,90 @@ echo
 echo
 sleep 2
 
+function compile-TPccminer {
+  echo "Compiling tpruvot ccminer"
+  echo " This could take a while ..."
+  if [[ $CUDA_VER == "8.0" ]]
+  then
+    if [[ -d ${NVOC_MINERS}/TPccminer/src/ ]]
+    then
+      cd ${NVOC_MINERS}/TPccminer/src/ 
+      if [[ $(git config --get remote.origin.url) != $TPccminer_repo ]]
+      then
+        echo "Wrong repo, clean up"
+        rm -rf ${NVOC_MINERS}/TPccminer/src
+        echo "Clone tpruvot repo from $TPccminer_repo"
+        git clone $TPccminer_repo ${NVOC_MINERS}/TPccminer/src/
```

Comment:
It is better to register the sub-repo as submodule for tracking since it is inside another one the pure clone creates problem to the upper level repo when upgrading miners. You also loose version reporting  in nvOC report command for non-submodule repos. If you see how pluggable-compiler is implemented in my branch you can see it creates a submodule if needed.

Reply:
This clone command will also fetch te whole repo history and it is likely to take a very long amount of time.